### PR TITLE
update usage text for tv11 and xgp11

### PR DESCRIPTION
**Pull request by n7275**

### CODE REVIEW

File: tv11.c [hunk 1]
```diff
@@ -333,6 +333,8 @@ usage(void)
 	fprintf(stderr, "\tserver: host running pdp-10\n");
 	fprintf(stderr, "\t-p: pdp-10 port; default 1110\n");
 	fprintf(stderr, "\t-l: tv11 listen port; default 11100\n");
+	fprintf(stderr, "\t-s: sleep (ms); detault=0\n");
```
Comment (larsbrinkhoff) [inline tv11.c]: Typo; should be default.  Also in xgp11.c.

File: xgp11.c [hunk 2]
```diff
@@ -328,7 +328,7 @@ char *argv0;
 void
 usage(void)
 {
-	fprintf(stderr, "usage: %s [-p port] server\n", argv0);
+	fprintf(stderr, "usage: %s [-p port] [-s time(ms)] [-d] server\n", argv0);
```
Comment (larsbrinkhoff) [inline xgp11.c:331]: Please squash into previous commit.

**Review by larsbrinkhoff — CHANGES_REQUESTED** [review]

File: tv11.c (hunk 1 above)
Comment (n7275) [reply tv11.c]: this should be fixed now

File: xgp11.c (hunk 2 above)
Comment (n7275) [reply xgp11.c:331]: also fixed

**Pull request merged**